docs/faq.rst

```diff
@@ -123,3 +123,22 @@ Yes, these techniques are complementary.
 Multiband fMRI leads to collecting multiple slices within a volume  simultaneously, while multi-echo
 fMRI is instead related to collecting multiple unique volumes.
 These techniques can be combined to reduce the TR in a multi-echo sequence.
+
+********************************************************************************
+[ME-fMRI] How do field maps and distortion correction work with multi-echo fMRI?
+********************************************************************************
+
+There are many different approaches to susceptibility distortion correction out there- far too many to discuss here.
+However, the good news is that distortion does not typically change across echoes in multi-echo fMRI.
+In most cases, the ``EffectiveEchoSpacing`` is the same across echoes,
+so distortion will remain relatively constant, even as dropout increases with echo time.
+
+What this means is that, in the vast majority of multi-echo datasets,
+standard distortion correction methods will work, and should be applied in the same manner on all echoes.
+For example, if you acquire a blip-up/blip-down set of images for all of your echo times,
+you should use the first echo time's images to generate the undistortion transform,
+as long as that first echo has sufficient gray/white constrast to be useful for alignment
+(in which case, use the earliest echo that does have good contrast).
+
+For context, please see
+`this NeuroStars thread <https://neurostars.org/t/multi-echo-pepolar-fieldmaps-bids-spec-sdcflows-grayzone/23933/5>`_.
```

Comment:
I like the idea of starting to link to NeuroStars from our docs so that people know to go there with questions and we don't have to reword entire discussions with examples in the docs.